src/bsp/platform/stm32f4_nucleo_wifi/xi_bsp_time_stm32f4_nucleo_wifi.c

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2003-2016, LogMeIn, Inc. All rights reserved.
+ *
+ * This is part of the Xively C Client library,
+ * it is licensed under the BSD 3-Clause license.
+ */
+
+#include <xi_bsp_time.h>
+
+void xi_bsp_time_init()
+{
+}
+
+xi_time_t xi_bsp_time_getcurrenttime_seconds()
+{
+    return 0;
```

Comment:
we just received a response from the st guys and there is a way to get time from the wifi board. we should put it to another story.